scripts/git/pre-commit.linter_python

```diff
@@ -72,12 +72,18 @@ echo "  Analysing python code style with 'isort'."
 # could happen for example if the .isort.cfg src_paths list gets out
 # of date.
 
-if $(isort ${py_changed} --profile black --diff --check-only --quiet >/dev/null); then
+isort_args=(
+    "--extend-skip=cmake"
+    "--extend-skip=submodules"
+    "--extend-skip=astrobee/survey_manager/survey_dependencies"
+    "--profile=black"
+)
+if $(isort . "${isort_args[@]}" --diff --check-only --quiet >/dev/null); then
```

Comment:
It's important to use `.` here as explained in the comment above and not restrict `isort` to looking at changed files. The `--extend-skip` stuff is to avoid linting external code.

Note this is following how it's done in the `astrobee` repo, except in my local testing I found that you need to use multiple `--extend-skip` arguments instead of a single arg with a comma-separated list.